source/deployment/on-boarding.rst

```diff
@@ -25,7 +25,7 @@ Creating System Admin account from the command line
 
 Deactivating a user
   - System Admins can go to **System Console > Users** for a list of all users on the server. The list can be searched and filtered to make finding the user easier. Click the user's role and in the menu that opens, click **Deactivate**.
-  - To preserve audit history, users are never deleted from the system. It is highly recommended that System Admins do not attempt to delete users manually from the database, as this may compromise system integrity and the ability to upgrade in the future.
+  - To preserve audit history, users are typically never deleted from the system. If permanently deleting a user is necessary (e.g. for the purposes of GDPR), a :doc:`CLI tool <../administration/command-line-tools>` can be used to do so.
```

Comment:
Should we spell out GDPR? (General Data Protection Regulation)

Reply:
Good point - I added a link for it, since `General Data Protection Regulation` may not be clear either though.